pyanaconda/modules/storage/storage.py

```diff
@@ -177,6 +178,14 @@ def setup_kickstart(self, data):
         if self.applied_partitioning:
             self.applied_partitioning.setup_kickstart(data)
 
+    def generate_kickstart(self):
+        """Generate kickstart string representation of this module's data
+
+        Adds Blivet version to the output because most of the strings come from Blivet anyway.
+        """
+        return "# Generated using Blivet version {}\n".format(blivet_version) + \
+               super().generate_kickstart()
```

Comment:
Please, add this string to the format above to avoid the concatenation.

Reply:
Done...